### PR TITLE
Stop duplicate contractor comments

### PR DESCRIPTION
**Pull request by singingwolfboy**
@sarina

### CODE REVIEW

File: openedx_webhooks/github_views.py [hunk 1]
```diff
@@ -455,6 +460,23 @@ def github_contractor_pr_comment(pull_request):
...
+    my_username = me["login"]
+    comment_url = "/repos/{repo}/issues/{num}/comments".format(
+        repo=pull_request["base"]["repo"]["full_name"].decode('utf-8'),
+        num=pull_request["number"],
+    )
+    for comment in paginated_get(comment_url, session=github):
+        # I only care about comments I made
+        if comment["user"]["login"] != my_username:
+            continue
+        magic_phrase = "It looks like you're a member of a company that does contract work for edX."
```
Comment (nedbat) [inline openedx_webhooks/github_views.py]: Is there any way to share this text with the code that makes the comment, so they won't get out of sync?

File: openedx_webhooks/github_views.py [hunk 2]
```diff
@@ -455,6 +460,23 @@ def github_contractor_pr_comment(pull_request):
     )
 
 
+def has_contractor_comment(pull_request, session=None):
```
Comment (nedbat) [inline openedx_webhooks/github_views.py]: As this code gets more complex, it would be helpful to write docstrings.

Comment (singingwolfboy) [inline openedx_webhooks/github_views.py]: Added.

File: openedx_webhooks/github_views.py (hunk 1 above)
Comment (singingwolfboy) [inline openedx_webhooks/github_views.py]: I don't think there's a way to share the text, unfortunately. But we can write tests!

Comment (sarina) [inline openedx_webhooks/github_views.py]: Yeah I'm concerned about this

Comment (sarina) [inline openedx_webhooks/github_views.py]: Alternatively, a simple test that tests that `magic_phrase` is in the contractor comment, so if we change the contractor comment the test would flag this?

Comment (singingwolfboy) [inline openedx_webhooks/github_views.py]: I added a test that asserts that after the bot has made a comment, the `has_contractor_comment` function returns True.

**Comment by sarina** [comment]
:+1: once tests pass

**Pull request merged**